test/common/testing_idp.go

```diff
@@ -332,7 +332,7 @@ func createClientSecret(ctx context.Context, client dynclient.Client, clientSecr
 }
 
 // create rhmi developer keycloak users
-func createKeycloakUsers(ctx context.Context, client dynclient.Client, installationName string) error {
+func createKeycloakUsers(ctx context.Context, client dynclient.Client, installation *integreatlyv1alpha1.RHMI, defaultNumberOfTestUsers int, testingIDPRealm string) error {
```

Comment:
I don't have a strong opinion on it but prefer leaving it as rhmi object rather than throwing strings around. Since this is not breaking/affecting anything I would rather leave it as it is.

Reply:
Agreed to in point above